src/qgraphicsmozview_p.cpp

```diff
@@ -540,7 +541,12 @@ void QGraphicsMozViewPrivate::OnLocationChanged(const char* aLocation, bool aCan
     if (mLocation != aLocation) {
         mLocation = QString(aLocation);
         mViewIface->urlChanged();
```

Comment:
Right. The first version of the patch emitted locationChanged unconditionally. With the commit https://github.com/nemomobile-packages/gecko-dev/commit/85542a6a948713229e56bb7b0cdb5ad7bfda505f the need for unconditional signal was removed, but forgot about existence of urlChanged.

Yesterday the commit was reverted, so I need to test the whole feature again. Please don't merge.